$OpenBSD: patch-src_gui_qgsmapcanvas_h,v 1.1 2013/11/08 22:40:38 sthen Exp $

Adapted from commit 641359d3cb70706f367014648cd961e905bca2dd
Author: Christian Urich <christian.urich@gmail.com>
Date:   Tue Sep 10 12:21:08 2013 +1000

Fix with newer sip

--- src/gui/qgsmapcanvas.h.orig	Wed Feb 22 20:48:14 2012
+++ src/gui/qgsmapcanvas.h	Fri Nov  8 10:35:47 2013
@@ -389,6 +389,11 @@ class GUI_EXPORT QgsMapCanvas : public QGraphicsView
     /// Handle pattern for implementation object
     std::auto_ptr<CanvasProperties> mCanvasProperties;
 
+    /**debugging member
+       invoked when a connect() is made to this object
+    */
+    void connectNotify( const char * signal );
+
   private:
     /// this class is non-copyable
     /**
@@ -432,11 +437,6 @@ class GUI_EXPORT QgsMapCanvas : public QGraphicsView
        rendering may put their sizes into this list. The canvas then picks up
        the last entry in case a lot of resize events arrive in short time*/
     QList< QPair<int, int> > mResizeQueue;
-
-    /**debugging member
-       invoked when a connect() is made to this object
-    */
-    void connectNotify( const char * signal );
 
     //! current layer in legend
     QgsMapLayer* mCurrentLayer;
